Makefile

```diff
@@ -59,10 +59,10 @@ SPACE             := $(EMPTY) $(EMPTY)
 smallify = $(subst $(SPACE),-,$(call lc,$1))
 
 # Naming for binaries and packaging etc,.
-OP_BIG_NAME := LibrePilot
-GCS_BIG_NAME := ${OP_BIG_NAME} GCS
+ORG_BIG_NAME := LibrePilot
+GCS_BIG_NAME := ${ORG_BIG_NAME}-GCS
```

Comment:
I did, but maybe my logic was wrong. The GCS name is librepilot-gcs, so this is what I expected GCS_SMALL_NAME to be, hence the change to the BIG_NAME. I'll need to look at the desktop file sed line again in Linux.mk